src/about.js

```diff
@@ -0,0 +1,78 @@
+// eslint-disable-next-line no-unused-vars
+const pageAbout = () => {
+  const canvas = document.querySelector('.canvas');
+
+  const about = document.createElement('DIV');
+  const ancestor = document.createElement('DIV');
+
+  about.classList.add('about');
+  ancestor.classList.add('tile');
+  ancestor.classList.add('is-ancestor');
+
+  canvas.appendChild(about);
+  about.appendChild(ancestor);
+
+  // eslint-disable-next-line no-unused-vars
+  const image = (() => {
+    const parent = document.createElement('DIV');
+    const child = document.createElement('DIV');
+    const figure = document.createElement('FIGURE');
+    const img = document.createElement('IMG');
+
+    parent.classList.add('tile');
+    parent.classList.add('is-parent');
```

Comment:
[Optional] - `classList` can take multiple classes separated by coma.
E.g `parent.classList.add('tile','is-parent')`